FruitFruit/FruitFruit/Utilities/Storage.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Storage.swift
+//  FruitFruit
+//
+//  Created by Junyeong Park on 2022/07/19.
+//
+
+import Foundation
+
+struct Storage {
+    static let defaults = UserDefaults.standard
```

Comment:
```suggestion
    static let userDefaults = UserDefaults.standard
```

defaults가 어떤 기본인지 알기 힘든 것 같아요 ~